python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/registry.py

```diff
@@ -183,8 +183,8 @@ def uppercase_first_letter(s):
         return op_name, cpp_class_name
 
     def _get_function_signature(self, function_kind: str,
-                                parameter_decl_builder: Callable["SIG_ATTR_TYPE", str],
-                                ret_decl_builder: Callable["SIG_ATTR_TYPE", str]) -> str:
+                                parameter_decl_builder: Callable[["SIG_ATTR_TYPE"], str],
+                                ret_decl_builder: Callable[["SIG_ATTR_TYPE"], str]) -> str:
```

Comment:
@ramiro050 I was getting the following error so I made these changes:
```
TypeError: Callable[args, result]: args must be a list. Got SIG_ATTR_TYPE
```
Do you also see these errors, or do I have stale code?

Reply:
Ah, this might be a Python version issue. Your version seems to be the right one.

Reply:
Hmm interesting. I am on python version `3.8.10`.

Reply:
There's probably something different about or Pythons. Mine never type checks when I execute a program, so that's why I didn't notice that error. Your implementation matches the docs for `Callable` so lets keep yours